src/main/java/com/team4/testingsystem/entities/Question.java

```diff
@@ -0,0 +1,83 @@
+package com.team4.testingsystem.entities;
+
+import javax.persistence.*;
+
+@Entity
+public class Question {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @Column(name = "question_body")
+    private String questionBody;
+    private int points;
+    @Column(name = "is_avaliable")
+    private int isAvailable;
```

Comment:
Why do "isAvailable" is an integer? Do we really have lots of reasons why do the question is unavailable and want to store it in a such way?
BTW. Field "isAvailable" with getters and setters like getIsAvailable/setIsAvailable seems weird. How do you think, can we use Boolean available with isAvailable(getAvailable) and setAvailable methods?

Reply:
it happened due to the fact that the isAvailable field in the database has the int type, that's why I wrote int here, do I need to fix this in the database?

P.s. of course the isAvailable method would be more logical

Reply:
Yep, I think it will be much more intuitive

Reply:
ok then I'll fix it in the database and make a PR to the repository with docs